multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/api/impl/FilesApiServiceImpl.java

```diff
@@ -81,6 +80,7 @@ public class FilesApiServiceImpl implements FilesApiService {
     private static final Duration HTTP_CONNECT_TIMEOUT = Duration.ofMinutes(10);
     private static final String RETRY_AFTER_SECONDS = "30";
     private static final String USERNAME_PASSWORD_URL_FORMAT = "{0}:{1}";
+
     static {
         System.setProperty(Constants.RETRY_LIMIT_PROPERTY, "0");
     }
```

Comment:
If the only changes to this class is reformatting, it think that they can be omitted.